include/rangers-api/Application/Text/AppTextListener.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+namespace app::text {
+    class AppTextListener : public hh::ui::TextListenerObject {
+        void* unkFromUnk1Param;
+        hh::text::ConverseTextListener* converseTextListener;
+        hh::text::TagReplacer* tagReplacer;
```

Comment:
Variable namings should be done in title case.
 Pointer types should be identified by a p before the variable name.